control-plane/stor-port/src/types/v0/transport/replica.rs

```diff
@@ -95,6 +95,19 @@ impl CreateReplicaSnapshot {
     }
 }
 
+/// The request type to delete a snapshot of a replica.
+pub struct DestroyReplicaSnapshot {
+    /// Incoming snapshot id to be deleted.
```

Comment:
```suggestion
/// The request type to delete a replica's snapshot.
pub struct DestroyReplicaSnapshot {
    /// Id of the snapshot to be deleted.
```

Reply:
Got this from the rebase.

Reply:
Rebase was not done correctly, see previous changes in the diff

Reply:
Done.